packages/core/src/Toggle/Toggle.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Toggle } from '.'
+import { render, fireEvent } from '../__test__/testing-library'
+
+describe('Toggle', () => {
+  test('renders correctly when off', () => {
+    const { getByRole } = render(<Toggle label='Total price toggle' />)
+    const container = getByRole('switch').parentNode
+    expect(container).toHaveStyleRule('background-color', '#4f6f8f')
```

Comment:
Can you pull the values used for the assertions from the theme?

Reply:
Great idea, updated to import colors from theme!